web/src/features/modals/InfoModal.tsx

```diff
@@ -31,7 +40,18 @@ export function InfoModalContent() {
         <VerticalDivider />
         <LegalNoticeButton />
       </div>
-      <p className="mt-2">{t('info.version', { version: APP_VERSION })}</p>
+      <div className="mt-2 flex items-center gap-2">
+        <p>{t('info.version', { version: APP_VERSION })}</p>
+        {needRefresh && (
+          <button
+            onClick={handleUpdate}
+            className="rounded bg-brand-green px-2 py-1 text-xs text-white hover:bg-brand-green/90"
+            title={t('updatePrompt.update')}
+          >
+            {t('updatePrompt.update')}
+          </button>
+        )}
+      </div>
```

Comment:
While this might be a good addition I've been using the service worker controls in the developer console (in chrome) to do this.

Reply:
The problem I find is when I'm trying to test on my mobile device

Reply:
Perhaps the settings modal would be more suited to that then?

I don't think we should be putting it in the info modal.

Reply:
I thought to put it here as it's right next to the version number and it only appears when an update is required, mostly as a dev feature.

Reply:
Makes sense but in the future I think we could have it as a non dev feature as well. To force updates when people are having issues.